@xen-orchestra/lite/src/App/TabConsole.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { withState } from 'reaclette'
+
+import Button from '../components/Button'
+import Console from '../components/Console'
+
+interface ParentState {}
+
+interface State {
+  ctrlAltDel: () => void
+}
+
+interface Props {
+  vmId: string
+}
+
+interface ParentEffects {}
+
+interface Effects {
+  setCtrlAltDel: (fn: () => void) => void
+}
+
+interface Computed {}
+
+const TabConsole = withState<State, Props, Effects, Computed, ParentState, ParentEffects>(
+  {
+    initialState: () => ({
+      ctrlAltDel: () => {},
+    }),
+    effects: {
+      setCtrlAltDel: function (fn) {
+        this.state.ctrlAltDel = fn
+      },
+    },
+  },
+  ({ effects, state, vmId }) => (
+    <div style={{ height: '100vh' }}>
+      <Button label='CTRL+ALT+DEL' onClick={state.ctrlAltDel} />
```

Comment:
```suggestion
      <Button label='Ctrl+Alt+Del' onClick={state.ctrlAltDel} />
```
Also, you have an internationalized message for this.